File: files-app/src/androidTest/java/l/files/ui/preview/RectCacheTest.java
package l.files.ui.preview;

import l.files.common.graphics.Rect;

public final class RectCacheTest
    extends PersistenceCacheTest<Rect, RectCache> {

  @Override RectCache newCache() {
    return new RectCache();
  }

  @Override Rect newValue() {
    return Rect.of(
        random.nextInt(100) + 1,
        random.nextInt(100) + 1);
  }
}

File: ui-preview/src/androidTest/java/l/files/ui/preview/RectCacheTest.kt
package l.files.ui.preview;

import l.files.ui.base.graphics.Rect;

public final class RectCacheTest extends PersistenceCacheTest<Rect, RectCache> {

    @Override
    RectCache newCache() {
        return new RectCache(mockCacheDir());
    }

    @Override
    Rect newValue() {
        return Rect.of(
                random.nextInt(100) + 1,
                random.nextInt(100) + 1);
    }
}
